src/components/CreateSpotAndPersona/CreateCard/CreateCard.tsx

```diff
@@ -37,7 +41,8 @@ const initialState: ImageRef = {
   blob: null,
 };
 
-export const CreateCard: FC = () => {
+export const CreateCard: FC<LinkProps> = (props: LinkProps) => {
```

Comment:
using FC<LinkProps> makes specifying type for argument redundant

Reply:
done